torchbox/torchbox.com
# Values block - BE and FE

[Link to FE Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1414650648)
[Link to BE Ticket](https://torchbox.monday.com/boards/1192293412/pulses/1414777233)

### Description of Changes Made

Adds the values block needed on the About us page - note we are using the service page type for this page.

### How to Test

In the pattern library: http://localhost:8000/pattern-library/render-pattern/patterns/molecules/streamfield/blocks/values_block.html

Or edit a service page in your local build, and choose to add a 'values' block

Note that I have not used desaturation for the images in this block.

### Screenshots

<details>
  <summary>Expand to see more</summary>

Page editor:
![Screenshot 2024-03-05 at 15 23 29](https://github.com/torchbox/torchbox.com/assets/771869/d8136fc8-3f64-4fe3-8a8d-f773382d72b7)

![Screenshot 2024-03-05 at 15 18 18](https://github.com/torchbox/torchbox.com/assets/771869/51a4cf35-ebfa-4dd4-8a78-db8f6afa4937)
![Screenshot 2024-03-05 at 15 17 08](https://github.com/torchbox/torchbox.com/assets/771869/7cf087b7-9f1f-4edd-834f-4944b14e5b70)

</details>

### MR Checklist


#### Unit tests


#### Documentation


#### Browser testing

  - Latest version of Chrome on mac
  - Latest version of Firefox on mac
  - Latest version of Safari on mac
  - Safari on last two versions of iOS
  - Chrome on last two versions of Android

#### Data protection


#### Accessibility


#### Sustainability


#### Pattern library
